burn-book/src/getting-started.md

```diff
@@ -199,14 +199,86 @@ For the sake of simplicity, the subsequent chapters of this book will all use th
 
 </div>
 
-## Running examples
+## Explore examples
+
+In the [next chapter](./basic-workflow) you'll have the opportunity to implement the whole Burn
+`guide` example yourself in a step by step manner.
 
 Many additional Burn examples are available in the
-[examples](https://github.com/tracel-ai/burn/tree/main/examples) directory. To run an example, please refer
-to its README.md for the specific command to execute.
+[examples](https://github.com/tracel-ai/burn/tree/main/examples) directory. Burn examples are
+organized as library crates with one or more examples that are executable binaries. An example
+can then be executed using the following cargo command line in the root of the Burn repository:
+
+```bash
+cargo run --example <example name>
+```
+
+To learn more about crates and examples, read the Rust section below.
+
+<details>
+<summary><strong>🦀 About Rust crates</strong></summary>
+
+Each Burn example is a **package** which are subdirectories of the `examples` directory. A package
+is composed of one or more **crates**.
+
+A package is a bundle of one or more crates that provides a set of functionality. A package
+contains a `Cargo.toml` file that describes how to build those crates.
+
+A crate is a compilation unit in Rust. It could be a single file, but it is often easier to
+split up crates into multiple **modules**.
+
+A module lets us organize code within a crate for readability and easy reuse. Modules also allow
+us to control the _privacy_ of items. For instance the `pub(crate)` keyword is employed to make
+a module publicly available inside the crate. In the snippet below there are four modules declared,
+two of them are public and visible to the users of the crates, one of them is public inside the crate
+only and crate users cannot see it, at last one is private when there is no keyword.
+These modules can be single files or a directory with a `mod.rs` file inside.
+
+```
+pub mod data;
+pub mod inference;
+pub(crate) mod model;
+mod training;
+```
+
+A crate can come in one of two forms: a **binary crate** or a **library crate**. When compiling a crate,
+the compiler first looks in the crate root file (`src/lib.rs` for a library crate and `src/main.rs`
+for a binary crate). Any module declared in the crate root file will be inserted in the crate for
+compilation.
+
+All Burn examples are library crates and they can contain one or more executable examples that
+uses the library. We have even some Burn examples that uses the library crate of other examples.
```

Comment:
Weird formulation: We have even -> We even have

Reply:
Fixed